src/Common/src/Interop/Unix/System.Native/Interop.MountPoints.FormatInfo.cs

```diff
@@ -143,91 +191,154 @@ private static DriveType GetDriveType(string fileSystemName)
                 case "minix":
                 case "minix_old":
                 case "minix2":
                 case "minix2v2":
+                case "minix3":
+                case "mlfs":
                 case "msdos":
+                case "nilfs":
+                case "nsfs":
+                case "ntfs":
                 case "ocfs2":
                 case "omfs":
                 case "openprom":
                 case "overlay":
                 case "overlayfs":
-                case "ntfs":
+                case "pstorefs":
                 case "qnx4":
+                case "qnx6":
                 case "reiserfs":
+                case "rpc_pipefs":
+                case "selinux":
+                case "smackfs":
                 case "squashfs":
                 case "swap":
+                case "sysfs":
                 case "sysv":
+                case "sysv2":
+                case "sysv4":
+                case "tracefs":
                 case "ubifs":
                 case "udf":
                 case "ufs":
                 case "umsdos":
                 case "umview-mod-umfuseext2":
+                case "usbdevfs":
+                case "v9fs":
+                case "vzfs":
+                case "wslfs":
+                case "xenfs":
                 case "xenix":
                 case "xfs":
+                case "xia":
                 case "xiafs":
                 case "xmount":
+                case "zfs":
                 case "zfs-fuse":
+                case "zsmallocfs":
                     return DriveType.Fixed;
 
                 case "9p":
-                case "autofs":
+                case "acfs":
+                case "afs":
+                case "aufs":
                 case "autofs4":
                 case "beaglefs":
+                case "ceph":
                 case "cifs":
                 case "coda":
                 case "coherent":
                 case "curlftpfs":
                 case "davfs2":
                 case "dlm":
+                case "fhgfs":
                 case "flickrfs":
+                case "fuseblk":
                 case "fusedav":
                 case "fusesmb":
+                case "gfsgfs2":
+                case "gfs/gfs2":
                 case "gfs2":
                 case "glusterfs-client":
                 case "gmailfs":
+                case "gpfs":
+                case "ibrix":
+                case "k-afs":
                 case "kafs":
                 case "ltspfs":
+                case "lustre":
                 case "ncpfs":
                 case "nfs":
                 case "nfs4":
+                case "nfsd":
+                case "novell":
                 case "obexfs":
+                case "panfs":
+                case "prl_fs":
                 case "s3ql":
                 case "smb":
+                case "smb2":
                 case "smbfs":
+                case "snfs":
                 case "sshfs":
-                case "sysfs":
```

Comment:
I agree this shouldn't be Network, but why Fixed?  Wouldn't it make more sense as RAM?